.github/workflows/e2e-test-manual.yml

```diff
@@ -245,7 +236,6 @@ jobs:
           gcpInClusterServiceAccountKey: ${{ secrets.GCP_CLUSTER_SERVICE_ACCOUNT }}
           test: ${{ inputs.test }}
           kubernetesVersion: ${{ inputs.kubernetesVersion }}
-          keepMeasurements: ${{ inputs.keepMeasurements }}
```

Comment:
the manual e2e has to set `fetchMeasurements` if a debug image is used. Otherwise PCR9 is taken from the embedded nightly image, instead of fetching fresh images for the debug image.